terraform/variables.tf

```diff
@@ -0,0 +1,34 @@
+variable "storage_account_name" {
+  type = string
+}
+
+variable "container_name" {
+  type = string
+}
+
+variable "resource_group_name" {
+  type = string
+}
+
+variable "resource_group_location" {
+  type = string
+}
+
+variable "key" {
+  type = string
+}
+
+variable "location" {
+  type    = string
+  default = "westeurope"
```

Comment:
I think this should be uksouth

Reply:
Yes, as we are trying to keep resources as close as possible to users (UK residents), this would ideally be a UK location.
This [Terraform region mapping list](https://github.com/claranet/terraform-azurerm-regions/blob/master/REGIONS.md) (from 3rd party, non-official source) does suggest `uksouth` as a valid option.
Has this been tried?

Reply:
We got [this error](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/runs/7148636612?check_suite_focus=true#step:8:38) when using uksouth in the end, so we changed it back to westeurope.